services/121-service/test/payment/payment-create-and-start.test.ts

```diff
@@ -131,65 +133,162 @@ describe('Start and create a payment separately', () => {
     expect(registrationAfterStart!.paymentCount).toBe(1);
   });
 
-  it('should not start transactions of non-included registrations', async () => {
+  describe('process included registrations only on payment starts', () => {
```

Comment:
This describe is a bit confusing. See output of the test run:

  Start and create a payment separately
    ✓ should set registration to complete on payment start when maxPayments is reached (3763 ms)
    ✓ should not facilitate 2nd payment when no created transactions for included registrations left (3095 ms)
    process included registrations only on payment starts
      ✓ should not start transactions of non-included registrations (3967 ms)
      ✓ should facilitate 2nd payment start when created transactions present (5442 ms)

Maybe something like just `payment start`